session/bootstrap.go

```diff
@@ -1000,6 +1000,13 @@ var internalSQLTimeout = owner.ManagerSessionTTL + 15
 // whether to run the sql file in bootstrap.
 var runBootstrapSQLFile = false
 
+// DisableRunBootstrapSQLFileInTest only used for test
+func DisableRunBootstrapSQLFileInTest() {
```

Comment:
Maybe move it to another file is better. ( no need reviewer by `sig-critical-approvers-tidb-server`